### DISCUSSION TITLE
Wrapper for aws sdk v3

### DISCUSSION THREAD

**Discussion by taschmidt** [KyleRoss/node-lambda-log#48] [Show and tell]
I wrote this to help myself out and thought maybe others might find it useful. It's very small and crude but it got the job done:

https://github.com/taschmidt/aws-lambda-log-wrapper

**Comment by KyleRoss** [comment]
This is a great idea! I'm curious if there is some way we can collaborate on this in the near future and integrate this into the core of LambdaLog itself?

**Comment by taschmidt** [comment]
Absolutely! Let me know whatever you think makes the most sense.
